pkgs/development/libraries/leveldb/default.nix

```diff
@@ -15,7 +15,12 @@ stdenv.mkDerivation rec {
     make all leveldbutil libmemenv.a
   '';
 
-  installPhase = "
+  installPhase = (stdenv.lib.optionalString stdenv.isDarwin ''
```

Comment:
FYI. there's a `fixDarwinDylibNames` helper that basically does this for you.

Reply:
Oh! Thank you! I actually knew about this, but could not remember how to find that name. I will try to update now.

Reply:
@LnL7 
It doesn't quite work for libleveldb:
```
% otool -D $(nix-build -A leveldb .)/lib/lib*dylib*
/nix/store/d1hyikxwzcvzbqj9x6mqqsf9xs91lbj2-leveldb-1.18/lib/libleveldb.dylib:
/nix/store/d1hyikxwzcvzbqj9x6mqqsf9xs91lbj2-leveldb-1.18/lib/libleveldb.dylib
/nix/store/d1hyikxwzcvzbqj9x6mqqsf9xs91lbj2-leveldb-1.18/lib/libleveldb.dylib.1:
/private/var/folders/1f/2mj_ng1n2y3_wc59fgw3fn_r0000gn/T/nix-build-leveldb-1.18.drv-0/source/libleveldb.dylib.1
/nix/store/d1hyikxwzcvzbqj9x6mqqsf9xs91lbj2-leveldb-1.18/lib/libleveldb.dylib.1.18:
/private/var/folders/1f/2mj_ng1n2y3_wc59fgw3fn_r0000gn/T/nix-build-leveldb-1.18.drv-0/source/libleveldb.dylib.1
```

Reply:
It is not correct that libleveldb.dylib, libleveldb.dylib.1, libleveldb.dylib.1.18 are just copies and not symlinks. I propose to however use my original fix. This is because the issue is temporary and will likely be gone by next release of leveldb that is going to use cmake.

Reply:
That's fine, just mentioned in case you didn't know about it 😄